packages/mdc-list/component.ts

```diff
@@ -129,6 +129,10 @@ export class MDCList extends MDCComponent<MDCListFoundation> {
     }
   }
 
+  setEnabled(itemIndex: number, isEnabled: boolean) {
```

Comment:
Add TSDoc for method description & params.

Reply:
Done!